src/main/kotlin/com/hellofresh/deblibs/slack/SlackMessage.kt

```diff
@@ -36,7 +36,7 @@ data class SlackMessage(
         enum class Color(val hexColor: String) {
             RED("#c4291c"),
             GREEN("#36a64f"),
-            YELLOW("#d6a048")
+            YELLOW("#d6a048"),
```

Comment:
Is this last comma valid? I mean, doesn't it throw an error at compile time?
```suggestion
            YELLOW("#d6a048")
```

Reply:
Oh, it's valid. It compiles. I read somewhere it's a good thing to do as future proof. When we add another a new enum it doesn't affect changes

Reply:
Alright, thanks for the feedback. I'm approving the PR :smiley:

Reply:
No problem. Thanks for the review.